initial logging docu (#148)
rico-projects/rico: documentation/src/docs/asciidoc/logging/centralized-logging.adoc

ifndef::imagesdir[:imagesdir: ../images]

=== Centralized logging

A centralized logging infrastructure brings many advantages.
Amongst them are:

* seeing logs from all application hosts in a single location
* configurable UI
* advanced search tools
* stored queries
* alerting on special events

> For Greylog and ELK we offer docker-compose based sample setup which are perfect for development.

Reviewer: ```suggestion
For Greylog and ELK our samples contain a docker-compose based setup which are well-proven for development.
```
